main.py

```diff
@@ -120,24 +127,26 @@ def display_travel_route(
             name = name[:22] + "..."
         s += f"-> [{br_id}] {name}: {lat} {lon} Distance: {distance} km.\n"
 
-    s += f"<- HOME: {start_lat}, {start_lon} Distance: {distance_to_start} km.\n"
+    s += f"<- HOME: {start_lat}, {start_lon} Distance: {distance_to_start} km.\n\n"
     s += f"Total distance: {total_distance} km.\n"
     return s
 
 
 def display_beer_list(travel_df):
     """
-    Display a list of beer types collected on the travel route
+    Format a detailed list of beer types collected on the travel route
     """
-    print(f"Collected {count_beer(travel_df)} beer types:")
+    s = ""
+    s += f"Collected {count_beer(travel_df)} beer types:\n"
     for row in travel_df.index:
```

Comment:
Avoid using `for x in df.index`
If you need values from some columns just take the columns that you need without using index.